docs/agent/cli-api.mdx

```diff
@@ -4964,6 +4966,8 @@ ngrok api event-sources create [flags]
 
 | Flag                | Description                                                                             |
 | ------------------- | --------------------------------------------------------------------------------------- |
+| `--fields`          | TODO                                                                                    |
```

Comment:
Hmmm, someone needs to go into the code and document these or mark them as disabled/hidden for now. Not gonna hold this PR up for that though

Reply:
Good call, will do